llmfoundry/__init__.py

```diff
@@ -87,5 +87,3 @@
     'tokenizers',
     'utils',
 ]
-
-__version__ = '0.11.0.dev0'
```

Comment:
You need to reexport this here so it is still available as `llmfoundry.__version__` (https://github.com/mosaicml/composer/blob/722888eeb577804a4f6de2fae47fc38f42d9bcbd/composer/__init__.py#L6)

Reply:
Just exported it here.